tools/prkit/managed-path-guard.py

```
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+"""Validate one managed path without following links or reparse points."""
+
+from __future__ import annotations
+
+import argparse
+import ntpath
+import os
+import pathlib
+import stat
+import sys
+from typing import Callable
```

Ruff (0.15.21)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)